app/Http/Controllers/AuthController.php

```diff
@@ -32,8 +22,6 @@ public function register(RegisterRequest $request)  {
      return response()->json([
```

Comment:
do it in 1 line and use constants for response statuses:
return response()->json(['token' => $token], HttpResponse::HTTP_CREATED);